Fix GitHub pages index when using mmark. (#332)
martinthomson/i-d-template: doc/SETUP.md

## Other tools

> If you wish to build GitHub Pages, you will also need to install the `toml` Python package:

Reviewer: ```suggestion
If you wish to build GitHub Pages from markdown, you will also need to install
the `yaml` (for kramdown-rfc) and `toml` (for mmark) Python package:
```

I think that this has to be an "and" to avoid having the script complain when you invoke it.  You could conditionally import `toml` or `yaml` in the script, but this seems easiest.

Reviewer: Maybe just this instead:

```suggestion
If you wish to build GitHub Pages, you will also need to install the `toml` and `yaml` Python packages:
```

```sh

> $ pip3 install --user toml

Reviewer: ```suggestion
$ pip3 install --user toml yaml
```
